app.py

```
@@ -613,52 +821,153 @@ def integrations_coderabbit():
+                        "commits_analyzed": report_data.get('commits_analyzed', 0),
+                        "files_changed": report_data.get('files_changed', 0),
+                        "issues_found": report_data.get('issues_found', 0),
+                        "score": report_data.get('score', 'N/A'),
+                        "date_range": f"{from_date} to {to_date}",
+                        "timestamp": datetime.now().isoformat(),
+                        "raw_data": report_data
+                    }
+
+                    db.update_coderabbit_report(report_id, response_data)
+
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)